pkg/features/kube_features.go

```diff
@@ -1120,8 +1120,8 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 		{Version: version.MustParse("1.33"), Default: false, PreRelease: featuregate.Beta},
 	},
 
-	DeploymentPodReplacementPolicy: {
-		{Version: version.MustParse("1.32"), Default: false, PreRelease: featuregate.Alpha},
+	DeploymentReplicaSetTerminatingReplicas: {
+		{Version: version.MustParse("1.33"), Default: false, PreRelease: featuregate.Alpha},
```

Comment:
We have made a mistake in https://github.com/kubernetes/kubernetes/pull/128546 and merged an incorrect version for this FG